people/ben-cerveny/index.md

```diff
@@ -0,0 +1,13 @@
+---
+role: director
+---
+
+![ben cerveny](nebBio.jpg)
+
+# Ben Cerveny
+## Director
+
+Ben is co-founder of the Foundation for Public Code.   He has worked for over 20 years on user interaction and experience design, concept prototyping, and strategy in the context of media applications, operating systems, web services, ubiquitous computing, and massively multiplayer games. Most recently, he founded Teem, a technology development studio focusing on platforms for connected environments. Before that, he was founder of celebrated VC-backed startup Bloom, whose iPad app Planetary was the first code in the Smithsonian Museum’s design collection. He also served as concept designer and strategist at Stamen Design, a leading visualization design studio. He was the founder of the Experience Design Lab at frogdesign, an international product design company, and a lead game designer and platform development strategist at Ludicorp, makers of Flickr.
```

Comment:
Often it is easier to manage codebases where lines are not this long. I often like pressing enter after every period, which will be ignored by markdown. This enables you to give feedback per line or make changes with clear simple diffs.

⚠️Careful, there is a `   ` (or three spaces) after the first period. If you press enter after that it will render it as a newline since space-space-newline is the token for `<br>` in markdown.